src/stylesheets/components/_footer.scss

```diff
@@ -309,9 +343,16 @@
         justify-content: center;
         margin-right: units(0.5);
         margin-left: units(-0.5);
+
+        @media (forced-colors: active) {
+          background-color: buttonText;
+          background-image: initial !important;
```

Comment:
`background-image: initial !important;` doesn't seem to do anything. Let's remove if not necessary.

Reply:
Resolved in [70eb57e](https://github.com/uswds/uswds/pull/4528/commits/70eb57e99dd316fb0fe0058fcb72e5d5456cd66a)